Assets/Scripts/TorchManager.cs

```diff
@@ -6,14 +6,18 @@ public class TorchManager : MonoBehaviour
 {
     [SerializeField] private bool isTorchOn = false;
     [SerializeField] private Light TorchLight;
+    [SerializeField] private ParticleSystem fire_particles;
```

Comment:
using snake case in C# is a war crime
private variables should be in camelCase `fireParticles`.
pls fix `TorchLight` as well to `torchLight`